### PR TITLE
Use timestamp from Jetty as creation time for Request/HttpRequest

### PR DESCRIPTION
**Pull request by bjorncs**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: document/abi-spec.json [hunk 1]
```diff
@@ -3551,4 +3551,4 @@
       "protected com.yahoo.document.update.ValueUpdate$ValueUpdateClassID valueUpdateClassID"
     ]
   }
-}
+}
```
Comment (baldersheim) [inline document/abi-spec.json:3554]: eol at eof, perhaps update tool.....

File: documentapi/abi-spec.json [hunk 2]
```diff
@@ -3207,4 +3207,4 @@
     ],
     "fields" : [ ]
   }
-}
+}
```
Comment (baldersheim) [inline documentapi/abi-spec.json:3210]: eol at eof

File: hosted-zone-api/abi-spec.json [hunk 3]
```diff
@@ -133,4 +133,4 @@
     ],
     "fields" : [ ]
   }
-}
+}
```
Comment (baldersheim) [inline hosted-zone-api/abi-spec.json:136]: eol at eof

**Review by baldersheim — APPROVED** [review]

**Review by jonmv — COMMENTED** [review]
👍

**Pull request merged**